README.md

```diff
@@ -7,6 +7,8 @@
 
 [Read the profile](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/profile.md)
 
+[View the workflows](diagram.md)
```

Comment:
I wonder if we should have the appropriate part of the workflow added to the right section of the spec too? Or at least a pointer from the spec to the entire thing. Right now if you just read the spec you wouldn't even know the picture existed.

Reply:
I would +1 having the diagram at the bottom and linked to from the index